penguin/tools/editing/legacy_multifile.py

```
@@ -171,8 +154,14 @@ def apply_multiedit(
         except Exception:
             applied_paths: List[str] = []
             created_paths: List[str] = []
+            snapshots: Dict[str, bytes] = {}
             for edit in edits:
                 target_path = Path(edit.file_path)
+                if target_path.exists() and str(target_path) not in snapshots:
+                    try:
+                        snapshots[str(target_path)] = target_path.read_bytes()
+                    except Exception:
+                        pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)